Projects/Foundations/Schemas/Docs-Topics/H/Heikin/Heikin-Data-Mine/heikin-data-mine-003-heikin-ashi-supertrend-heikin-indicator.json

```diff
@@ -0,0 +1,222 @@
+{
+    "topic": "Heikin Data Mine",
+    "pageNumber": 3,
+    "type": "Heikin Ashi Supertrend - Heikin Indicator",
+    "definition": {
+        "text": "Produce the supertrend indicator using the Heikin Ashi Candles as a basis for its calcuations.",
+        "updated": 1636853769,
+        "translations": [
+            {
+                "language": "RU",
+                "text": "Создайте индикатор супертренда, используя свечи Heikin Ashi в качестве основы для своих расчетов.",
+                "updated": 1643521546610
+            },
+            {
+                "language": "DE",
+                "text": "Erstellen Sie den Supertrend Indikator, indem Sie die Heikin Ashi Candles als Grundlage für seine Berechnungen verwenden.",
+                "updated": 1644726217034
+            }
+        ]
+    },
+    "paragraphs": [
```

Comment:
Did you make the files that are shown completely green or did you just translate it into german?